perfdash/config.go

```diff
@@ -340,6 +340,45 @@ var (
 					Parser:           parsePerfData,
 				}},
 		},
+		"Network_Baseline": {
+			"TCP_P2P": []TestDescription{{
+				Name:             "network_performance_tcp-1:1",
+				OutputFilePrefix: "NetworkPerformanceMetrics_1:1_TCP_P2P",
+				Parser:           parsePerfData,
+			}},
+
+			"UDP_P2P": []TestDescription{
+				{
```

Comment:
Please make pagination consistent - just use the style you use above [i.e. {{ in the previous line]

same below

Reply:
Modified